ext/ftp/ftp.stub.php

```diff
@@ -57,63 +57,63 @@ function ftp_systype($ftp): string|false {}
 
 /**
  * @param resource $ftp
- * @param resource $fp
+ * @param resource $stream
  */
-function ftp_fget($ftp, $fp, string $remote_file, int $mode = FTP_BINARY, int $resumepos = 0): bool {}
+function ftp_fget($ftp, $stream, string $remote_filename, int $mode = FTP_BINARY, int $resume_position = 0): bool {}
 
 /**
  * @param resource $ftp
- * @param resource $fp
+ * @param resource $stream
  */
-function ftp_nb_fget($ftp, $fp, string $remote_file, int $mode = FTP_BINARY, int $resumpos = 0): int {}
+function ftp_nb_fget($ftp, $stream, string $remote_filename, int $mode = FTP_BINARY, int $resume_position = 0): int {}
 
 /** @param resource $ftp */
-function ftp_pasv($ftp, bool $pasv): bool {}
+function ftp_pasv($ftp, bool $enable): bool {}
 
 /** @param resource $ftp */
-function ftp_get($ftp, string $local_file, string $remote_file, int $mode = FTP_BINARY, int $resumepos = 0): bool {}
+function ftp_get($ftp, string $local_filename, string $remote_filename, int $mode = FTP_BINARY, int $resume_position = 0): bool {}
 
 /** @param resource $ftp */
-function ftp_nb_get($ftp, string $local_file, string $remote_file, int $mode = FTP_BINARY, int $resume_pos = 0): int {}
+function ftp_nb_get($ftp, string $local_filename, string $remote_filename, int $mode = FTP_BINARY, int $resume_position = 0): int {}
 
 /** @param resource $ftp */
 function ftp_nb_continue($ftp): int {}
 
 /**
  * @param resource $ftp
- * @param resource $fp
+ * @param resource $stream
  */
-function ftp_fput($ftp, string $remote_file, $fp, int $mode = FTP_BINARY, int $startpos = 0): bool {}
+function ftp_fput($ftp, string $remote_filename, $stream, int $mode = FTP_BINARY, int $start_position = 0): bool {}
```

Comment:
Why not use `$resume_position` or `$offset` here as well?

Reply:
I'm ok with `$offset` here too. Thanks for the idea!